config/server.lua

```diff
@@ -25,7 +25,7 @@ return {
         identifierTypes = {
             citizenid = {
                 valueFunction = function()
-                    return tostring(RandomLetter(3) .. RandomNumber(5)):upper()
+                    return tostring(lib.string.random('A', 3) .. lib.string.random('1', 5))
```

Comment:
```suggestion
                    return lib.string.random('........')
```
This isn't how this function works. Also why don't we just create 8 random alphanumerical characters instead of 3 letters and 5 numbers?